### DISCUSSION TITLE
Path concatenation with raw requests

### DISCUSSION THREAD

**Discussion by webtester0** [projectdiscovery/nuclei#3732] [Q&A]
The problem is this: when using raw syntax in the template, there is no possibility to specify strictly path, so when you run nuclei on URLs scope, the current path from template is concatenated with path which is passed in scope.
It's possible to specify Host strictly, just using annotation, but it's not possible with path

Is there any possible solution, to fix this problem?
<img width="1163" alt="image" src="https://github.com/projectdiscovery/nuclei/assets/44179403/f0ca4012-7cf1-451e-84b7-99b995d92e80">

**Comment by MetzinAround** [comment]
I see a quote mark after the `3` in `GET /api/v1/id=123` there. Is that affecting anything?

**Reply by webtester0** [reply]
No, it's just for detect simple SQLi
If I remove quote, I get the same results
<img width="736" alt="image" src="https://github.com/projectdiscovery/nuclei/assets/44179403/7dd60e67-1cda-4a7a-8186-ab5728650977">

**Reply by MetzinAround** [reply]
Thanks. 

@ehsandeep can you take a look when you have a chance?

**Reply by webtester0** [reply]
@MetzinAround @ehsandeep any updates?

**Comment by tarunKoyalwar** [comment]
@webtester0 , thanks for bringing this to our attention . i have created new issue for this at https://github.com/projectdiscovery/nuclei/issues/3764 . we will add new option/annotation to either disable concat behaviour by default or ignore for this template only

**Reply by webtester0** [reply]
@tarunKoyalwar thx

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
